specification/common-types/resource-management/v1/types.json

```diff
@@ -319,6 +319,110 @@
         }
       },
       "description": "The resource management error additional info."
+    },
+    "JSONWebKey": {
+      "type": "object",
+      "required": [
+        "use",
+        "kty",
+        "kid",
+        "alg"
+      ],
+      "properties": {
+        "alg": {
+          "description": "The \"alg\" (algorithm) parameter identifies the algorithm intended for\nuse with the key.  The values used should either be registered in the\nIANA \"JSON Web Signature and Encryption Algorithms\" registry\nestablished by [JWA] or be a value that contains a Collision-\nResistant Name.",
+          "type": "string",
+          "x-go-name": "Alg"
+        },
+        "crv": {
```

Comment:
It's quite confusing to have such naming pattern. Is it possible to use longer words? at least, we should add description.

Reply:
description are added